### ISSUE TITLE
Clicking links opens links in pop-out window, no option to change this behaviour

### ISSUE THREAD

**Issue by jrpeck1989** [getferdi/ferdi#1071]
In previous versions of Ferdi, left clicking on a link would open the link in an external browser. Now, it seems that every link opens in a pop up window. this is annoying, as a lot of links I click require me to be logged in, and therefore this popup window is of no use to me. 

OS: macOS 10.15.7
Ferdi version: 5.6.0-beta.2

**Comment by Judx** [comment]
I agree, I would like to continue it opening the link in an external browser instead of pop-up. Also pop-ups are not always working as they say "Grrr! A popup blocker may be preventing the application from opening the page..."

**Comment by Julian88Tex** [comment]
Still an issue and would love an option for always opening in external thanks!

**Comment by Julian88Tex** [comment]
Looks like this is a duplicate of #941

**Comment by Julian88Tex** [comment]
Also `CMD + Left Click` is a workaround
